workers/unity/Assets/Playground/Resources/Prefabs/Common/Cube.prefab

```diff
@@ -34,9 +24,10 @@ GameObject:
   m_IsActive: 1
 --- !u!4 &4711983361604210
 Transform:
-  m_ObjectHideFlags: 1
+  m_ObjectHideFlags: 0
   m_CorrespondingSourceObject: {fileID: 0}
-  m_PrefabInternal: {fileID: 100100000}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
```

Comment:
these changes: do we need to make them for the other projects as well?

Reply:
This happens automatically the first time you run play. So we can do that once things merge in